wolfssl/wolfio.h

```diff
@@ -491,6 +492,37 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
 
 #endif
 
+#ifdef WOLFSSL_GNRC
+    #include <sock_types.h>
+    #include <net/gnrc.h>
+    #include <net/af.h>
+    #include <net/sock.h>
+    #include <net/gnrc/tcp.h>
+    #include <net/gnrc/udp.h>
+
+    struct gnrc_wolfssl_ctx {
```

Comment:
Does this structure need to be in a header?  That is, if it's only used internally by wolfio.c couldn't it go in the .c file?

Reply:
This is the structure shared with RIOT-OS and exported (for the wolfssl RIOT-OS package) as `sock_tls_t`. See the API from our proposed wolfssl pkg in RIOT-OS:
https://github.com/danielinux/RIOT/blob/master/pkg/wolfssl/sock_tls/sock_tls.h